odh/base/monitoring/overrides/dashboards/jupyterhub-dashboard.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: integreatly.org/v1alpha1
```

Comment:
Do we need to add this file to some kustomization.yaml?

Reply:
I think we don't need to as the general overrides folder is already referred to in the kfdef file.